dhis-2/dhis-test-integration/src/test/java/org/hisp/dhis/maintenance/MaintenanceServiceTest.java

```diff
@@ -466,4 +486,12 @@ private boolean eventExistsIncludingDeleted(String uid) {
         jdbcTemplate.queryForObject(
             "select exists(select 1 from event where uid=?)", Boolean.class, uid));
   }
+
+  private EnrollmentOperationParams createEnrollmentParamsIncludingDeleted(String enrollmentUid) {
+    return EnrollmentOperationParams.builder()
```

Comment:
If you look at the code using this you can see that all usages actually want to do is to check if `enrollmentExistsIncludingDeleted`. So create `enrollmentExistsIncludingDeleted` like

```java 
private boolean enrollmentExistsIncludingDeleted(Enrollment enrollment)
```

This will make the tests more readable.